### ISSUE TITLE
meaning of `edge_length`, number of points

### ISSUE THREAD

**Issue by nschloe** [krober10nd/SeismicMesh#134]
Looking at the results of meshgen-comparison, it seems that seismicmesh has a different notion of `h` than other mesh generators. Check L-shape, cylinder, L-shape 3D. Any idea what's wrong?

https://github.com/nschloe/meshgen-comparison#l-shape

**Comment by krober10nd** [comment]
I noticed this too. As a sanity check, I produced a square with sides length 1 and specified an `edge_length` of 0.2, I get ~5 elements across. I repeated the below example with `h=0.1` and got 10 across. 

```
 import SeismicMesh
 
 import meshio
 
 h = 0.2
 
 rect = SeismicMesh.Rectangle((0.0, 1.0, 0.0, 1.0))
 points, cells = SeismicMesh.generate_mesh(
     domain=rect,
     edge_length=h,
 )
 meshio.Mesh(points, {"triangle": cells}).write("out.vtk")
```
![Simple](https://user-images.githubusercontent.com/18619644/98374516-89a06f80-201f-11eb-82b8-c37ea533475c.png)

I wonder what `cgal` and `gmsh` produce?

 I see you're multiplying the `max_edge_length` by 1.5? 

https://github.com/nschloe/meshgen-comparison/blob/ef896116acbe1a352b538a1037b6a90ba6206402/meshgen_comparison/pygalmesh_examples.py#L50

I assume this parameter name implies `cgal` and `gmsh` attempt to bound the *maximum* edge length, which I would assume implies all other edge lengths would be less than this.
